docker/local/django/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM python:3.7-slim-buster
+FROM python:3.8-slim-buster
```

Comment:
should this be changed? 🤔 (just "atirating uma laracha" 😂 )

Reply:
OPS! my bad, blame the M1...